.github/workflows/data_update_v2.yml

```diff
@@ -68,7 +73,13 @@ jobs:
         OUT_PATH="${GITHUB_WORKSPACE}/site/data/data.v2.json"
 
         python scraper_data.py > today.json 2> errors.txt || true
-        jq -s '.[0] + .[1]' $OUT_PATH today.json > $OUT_PATH
+
+        # Merge new data into previous data. (Note this has to be two steps;
+        # if we read $OUT_PATH as input for jq and write stdout to it at the
+        # same time, we get conflicts and bad output.)
+        jq -s '.[0] + .[1]' $OUT_PATH today.json > merged.json
```

Comment:
This is a question to original code, not your new code though, 
is there any reason why are we merging existing data.v2.json and new today.json instead of replacing existing one to new one? 

If it has to merge instead of replacing, the updated code looks good. Just curious why merging.

Reply:
Oh, I think I understand why. Some countries could fail and we want to keep the existing data for the counties.